index.html

```diff
@@ -2540,18 +2605,438 @@ <h1>
 Resolution
     </h1>
 
-    <p>
-A <a>DID resolver</a> is a software or hardware component with an API for
-resolving <a>DIDs</a> of at least one <a>DID method</a>. It executes the read
-operation for the <a>DID method</a> corresponding to the <a>DID</a> being
-resolved to obtain the authoritative <a>DID document</a>. For more information,
-see Section <a href="#read-verify"></a>.
+    <p class="info">
+This section defines the interfaces (or the "contracts") of the <a>DID resolution</a>
+and the <a>DID URL dereferencing</a> functions, including their inputs and results.
+For information about architectures, see Section <a href="#did-resolvers-and-did-resolution"></a>.
```

Comment:
I'm of the opinion that it's best to avoid discussing or referencing to architecture docs in this spec. I think that comes down to an implementation detail that an implementer could choose. Alternatively if we decide to have a "resolution implementation guide" then this detail would fit well in there.